Makefile

```diff
@@ -34,11 +40,14 @@ BASE_IMAGE_20H2=mcr.microsoft.com/windows/servercore:20H2
 # Both arrays MUST be index aligned.
 WINDOWS_IMAGE_TAGS=ltsc2019 1909 2004 20H2
 WINDOWS_BASE_IMAGES=$(BASE_IMAGE_LTSC2019) $(BASE_IMAGE_1909) $(BASE_IMAGE_2004) $(BASE_IMAGE_20H2)
+# Development only
```

Comment:
Did you mean to remove this before submitting?

Reply:
yeah, thanks for catching this, I've inlined these values later in the Makefile instead